language/lambda_spec.rb

```diff
@@ -274,6 +274,20 @@ def obj.define
     lambda { }.lambda?.should be_true
   end
 
+  it "requires a block" do
+    lambda { lambda }.should raise_error(ArgumentError)
+  end
+
+  context "with an implicit block" do
+    before do
+      def meth; lambda; end
+    end
+
+    it "can be created" do
+      meth { 1 }.().should == 1
```

Comment:
I think it might be worth to add an assertion that the returned lambda is in fact a lambda and not just the block transformed as a proc: `implicit_lambda.lambda?.should be_true`